proprietary/react-icons/README.md

```diff
@@ -1,5 +1,40 @@
 <!-- @license CC0-1.0 -->
 
-# Icons React
+# Amsterdam Design System: React icons
 
-Icon svgs as React components
+This package provides all icons from the [Amsterdam Design System](https://designsystem.amsterdam) as React components.
+Use it to use the correct icons in your React project.
+
+## Installation
+
+Install this package by itself if you want or need to use icons directly in your React project.
+
+Note that our [React components](https://www.npmjs.com/package/@amsterdam/design-system-react) contain an [Icon component](https://designsystem.amsterdam/?path=/docs/components-media-icon--docs) that offers a convenient API for using the icons and streamlines alignment with text.
+
+```sh
+npm install @amsterdam/design-system-react-icons
+```
+
+## Usage
+
+Import the component for the icon you need and use it in your JSX.
+
+```ts
+import { SearchIcon } from '@amsterdam/design-system-react-icons'
+
+export const App = () => (
+  <div>
```

Comment:
Seeing as we're already in a React project, maybe we should use Icon here as well?